pavelib/servers.py

```diff
@@ -135,6 +135,7 @@ def devstack(args):
     if args.optimized:
         settings = OPTIMIZED_SETTINGS
         asset_settings = OPTIMIZED_ASSETS_SETTINGS
+    sh(django_cmd('cms', settings, 'reindex_course', '--setup'))
```

Comment:
Is this the right place to put this? Why are you indexing on the cms and not the lms? I'm confused why the commands live in the CMS but the settings are in lms/envs/devstack.py.